core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -153,6 +153,7 @@
     public static final String QUERY_RETRY_ATTEMPTS = "query_retry_attempts";
     public static final String TASK_RETRY_ATTEMPTS_OVERALL = "task_retry_attempts_overall";
     public static final String TASK_RETRY_ATTEMPTS_PER_TASK = "task_retry_attempts_per_task";
+    public static final String MAX_TASKS_WAITING_FOR_NODE_PER_STAGE = "max_tasks_waiting_for_node_per_stage";
```

Comment:
it gets very long then :) Also session properties above are not prefixed - and also tardigrade specific .

Reply:
That's a good point. It looks like we are somehow inconsistent. Some of the properties are prefixed when some of them are not. I wonder what should be the guideline. What do you think?

Reply:
If property is only applicable to fault-tolerant execution I am option for prefixing. THough it would be nice to have something shorter than `fault_tolerant_execution_` :) No idea for alternative though.

On the other hand at some point we may migrate to just having a single execution mode - then prefixing is not needed. Then dropping prefix already is more future proof :)